scripts/integ-setup.bat

```diff
@@ -12,9 +12,9 @@ call npm run integ:templates
 
 :: install
 call lerna bootstrap
-call lerna add --scope integration-test aws-amplify
-call lerna add --scope integration-test @aws-amplify/ui-react
-call lerna add --scope integration-test @aws-amplify/datastore
+call lerna add --scope integration-test aws-amplify@^5.0.0
+call lerna add --scope integration-test @aws-amplify/ui-react@^5.0.0
+call lerna add --scope integration-test @aws-amplify/datastore@^5.0.0
```

Comment:
should this be `@aws-amplify/datastore@^4.0.0`?

Reply:
Yea, missed that one